common/parameters/Options.java

```diff
@@ -33,6 +35,7 @@
     private Boolean infer = null;
     private Boolean explain = null;
     private Integer batchSize = null;
+    private Optional<Boolean> prefetch = null;
```

Comment:
Unlike our other `Options`, `null` is considered a valid value for `prefetch`, and it means "whatever the default for this Graql query type is."

Reply:
the only comment: this is why we have `Boolean __ = null`, we don't store optionals :)

Reply:
Isn't that the semantics of every other field too? 🤔

Reply:
What I mean is it behaves differently to the others. The others all have DEFAULT values specified in this file. `prefetch` doesn't have a default - it defaults to whatever the default is for the Graql query that's being executed.

Reply:
I don't think this should be captured with optional